LEAF_Request_Portal/admin/templates/mod_svcChief.tpl

```diff
@@ -110,13 +111,23 @@ function getMembers(groupID) {
 
 function populateMembers(groupID, members) {
     $('#members' + groupID).html('');
+    var memberCt = (members.length - 1);
+    var countTxt = (memberCt > 0) ? (' + ' + memberCt + ' others') : '';
     for(var i in members) {
     	if(members[i].active == 1) {
-            $('#members' + groupID).append(members[i].Lname + ', ' + members[i].Fname + '<br />');
+            if (i == 0) {
+                $('#members' + groupID).append('<span>' + toTitleCase(members[i].Fname) + ' ' + toTitleCase(members[i].Lname) + countTxt + '</span>');
+            }
+            
     	}
     }
 }
 
+// convert to title case
+function toTitleCase(str) {
+    return str.replace(/\w\S*/g, function(txt){return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();});
+}
+
```

Comment:
This function seems useful - we should think about extracting functions like this into a helper file for rendering.